jvm/hermes/src/main/jni/JJSIValue.h

```diff
@@ -115,9 +112,9 @@ class JJSIValue : public JJSIValueHybridClass {
     static constexpr auto kJavaDescriptor = "Lcom/intuit/playerui/jsi/Value;";
     static void registerNatives();
 
-    static local_ref<jhybridobject> fromBool(alias_ref<jclass>, bool b);
-    static local_ref<jhybridobject> fromDouble(alias_ref<jclass>, double d);
-    static local_ref<jhybridobject> fromInt(alias_ref<jclass>, int i);
+    static local_ref<jhybridobject> fromBool(alias_ref<jclass>, alias_ref<JRuntimeThreadContext>, alias_ref<JJSIRuntime::jhybridobject> jRuntime, bool b);
+    static local_ref<jhybridobject> fromDouble(alias_ref<jclass>, alias_ref<JRuntimeThreadContext>, alias_ref<JJSIRuntime::jhybridobject> jRuntime, double d);
+    static local_ref<jhybridobject> fromInt(alias_ref<jclass>, alias_ref<JRuntimeThreadContext>, alias_ref<JJSIRuntime::jhybridobject> jRuntime, int i);
```

Comment:
Primitive `Value`s (applies to boolean, double, int) should also not need to be tracked because they're not backed by runtime data. Meaning, they shouldn't need either runtime thread context or the runtime args themselves.

This'd result in reverting the API changes for the primitive methods too, as they won't need the runtime thread context or runtime args.